src/main/java/digital/slovensko/autogram/ui/gui/SigningDialogController.java

```diff
@@ -76,7 +69,7 @@ public void onMainButtonPressed(ActionEvent event) {
         } else {
             gui.disableSigning();
             getNodeForLoosingFocus().requestFocus();
-            autogram.sign(signingJob, signingKey);
+            autogram.sign(visualization.getJob(), signingKey);
```

Comment:
@jsuchal toto je to co mi prislo trochu zvlastne (ze vizualizacia je zodpovedna za odovzdanie obsahu ktory sa ide podpisovat), ale vpodstate s tym nemam problem

Reply:
Ja toto beriem tak, ze mas signing job a potom mas "vizualizovany signing job". Nejako ma to nerusi. Skor mi smrdelo, ze sa vsade inak posielalo v parametroch job+visualization